### ISSUE TITLE
Sid Meier's Ace Patrol

### ISSUE THREAD

**Issue by jalefkowit** [ValveSoftware/Proton#1631]
# Compatibility Report
- Name of the game with compatibility issues: Sid Meier's Ace Patrol
- Steam AppID of the game: 244070

## System Information
- GPU: GTX 970
- Driver/LLVM version: NVidia 396.54
- Kernel version: 4.15.0-34
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/jalefkowit/9f3644c23e89ec35b06a8b0dc4a4e1e6
- Proton version: 3.7-6 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-244070.log](https://github.com/ValveSoftware/Proton/files/2429364/steam-244070.log)

## Symptoms 

Game never starts. Steam reports "Running..." in the list of games, but it just sits there for a minute or so and then the "Running..." label goes away.

## Reproduction

1. Double-click "Sid Meier's Ace Patrol" in list of games in Steam to launch

**Comment by thaewrapt** [comment]
Does this one also utilize CEG DRM?

**Comment by thaewrapt** [comment]
Yes it does, parent issue for reference: https://github.com/ValveSoftware/Proton/issues/753

Both Ace Patrol and Ace Patrol: Pacific Skies now start and are playable.

**Comment by jalefkowit** [comment]
I can't speak to whether these titles use CEG DRM, but they both still fail to start on my machine (which is different than the one I was using when I originally filed this issue three years ago). I've tried manually forcing them to use both Proton 6.3-7 and Proton Experimental, neither one works.

**Comment by thaewrapt** [comment]
@jalefkowit You need Steam Beta client as well, and it's for now working on Proton Experimental only.

Also, for CEG-enabled games it's showing something like "Steam servers are not available (or can't process request) at the moment" on a regular Proton before trying to update required dependencies on the first run, and if working (Steam Beta + Proton Experimental for now) it's gonna install DirectX, VC redistributables and stuff like that, then try to run the game.
